about.html

```diff
@@ -0,0 +1,166 @@
+        <h3 class="intro">"Hello! Ghana Gamers"</h3>
+        <h1 class="topic-tect">ACCRA <br> ESPORTS FESTIVAL 2023</h1>
+        <div class="divider"></div>
+        <div class="main-desc">
+            <p>
+            Our Elecronic Sports Festival brings together the broad community of Gamers who are willing to share knowledge, experience and skill. We hope that this Year extends interest in our industry beyond its members.
+            </p>
+            <p id="main-desc-long">
+                This festival also provides a platform for up-and-coming gamers to showcase their talents and potentially launch their careers. In addition to competition, we also offer workshops we call 'E-Talk' for attendees to learn from industry experts. We strive to promote inclusivity and diversity in the gaming community through our events and programming. We are excited to see the connections and friendships that will be formed at our festival, bringing the gaming community even closer together.            </p>
+            
+        </div>
+        <span class="email-contact">Please contact us per Email for any further questions about Accra Esports Festival 2023</span>
+        <a id="email" href="mailto:info@ghanaesportsfederation.org">info@ghanaesportsfederation.org</a>
+    </section>
+
+<!-- LOGO DETAILS------------------------------------------------------------------------------ -->
+    <section class="logo-details">
+        <h2 class="logo-details-topic">Accra Esports Festival 2023 Logo</h2>
+        <div class="divider"></div>
+        <p class="logo-desc hide-class">
+            Through the logo contest, which was held from November 8th to December 7th, the winning design was determined through online voting and evaluation by judges.The logo of Accra Esports Festival 2023 optimized for use on the homepage and promotional materials after consultation with the winning designer is as follows.
+        </p>
+        <p class="logo-desc-short">
+            The logo of Accra Esports Festival 2023 was slected as a winning design in a logo contest, which was held from November 8th to December 7th.
+        </p>
+        <div class="logo-box">
+            <img src="assets/logo.PNG" alt="" class="logo-logo">
+        </div>
+    </section>
+
+<!-- PAST EVENTS SECTION--------------------------------------------------------------------------- -->
+    <section class="past-events">
+        <h2 class="past-topic">See the Past Accra Esports Festivals</h2>
+        <div class="divider"></div>
+        <span>Take a look at the two Esports Festivals which took place in Kumasi and Tamale</span>
+        <div class="events-box">
+            <a href="#" class="event1">
+                <img src="assets/event1.png" alt="event1-img">
+            </a>
+            <a href="#" class="event1">
+                <img src="assets/event2.png" alt="event2-img">
+            </a>
+        </div>
+    </section>
```

Comment:
- [ ] **About Page:**
- Please follow the original design: The color of the pictures should look like in the original design
- See your design:
![Capture](https://user-images.githubusercontent.com/63972670/212201057-976d1a07-dcf1-4294-b823-ab851e4ec405.PNG)
- Original Design:
![Capture1](https://user-images.githubusercontent.com/63972670/212201210-78fb5ce1-e6d4-4a25-bdbb-c94e7b90acdd.PNG)

Reply:
All the changes requested have been made. Thanks @OLIPLICHE. 👌🏿